src/wfs/src/test/java/org/geoserver/wfs/GetFeatureTest.java

```diff
@@ -375,7 +377,25 @@ public void testRequestDisabledResource() throws Exception {
                 doc);
         XMLAssert.assertXpathEvaluatesTo("typeName", "//ogc:ServiceException/@locator", doc);
     }
-    
+
+    /**
+     * Test that a request for a resource from a disabled store fails.
+     */
+    @Test
+    public void testRequestDisabledStore() throws Exception {
+        Catalog catalog = getCatalog();
+        StoreInfo store = catalog.getStoreByName("cdf", DataStoreInfo.class);
+        store.setEnabled(false);
```

Comment:
So the store does not get re-enabled and the other tests pass anyways? Stroke of luck and it's one of the last tests to be run?